src/toxcore/dht/kbucket.rs

```diff
@@ -131,7 +158,8 @@ impl Bucket {
                 trace!("Creating a new Bucket with default capacity.");
                 Bucket {
                     capacity: BUCKET_DEFAULT_SIZE as u8,
-                    nodes: Vec::with_capacity(BUCKET_DEFAULT_SIZE)
+                    nodes: Vec::with_capacity(BUCKET_DEFAULT_SIZE),
+                    bad_node_timeout: Duration::default(),
```

Comment:
Should we set a preconfigured constant here?

Reply:
bad_node_timeout settting is done using this function 
https://github.com/tox-rs/tox/blob/45c6414fdd11c3c4fe7ada99aea981679885eaec/src/toxcore/dht/kbucket.rs#L326
And Bucket::new() is used many place in toxcore.
To have consistency, we have to use current new(Option<u8>) prototype.

Reply:
@kpp  was speaking about `bad_node_timeout`. Now we have ConfigArgs and set this value from these args. But actually it should be constant, and for now you can use this constant as default value instead of `Duration::default()`.

Reply:
> . But actually it should be constant, and for now you can use this constant as default value instead of Duration::default().

true

Reply:
done